packages/cicero-core/src/metadata.js

```diff
@@ -118,6 +119,10 @@ class Metadata {
             throw new Error('README must be a string');
         }
 
+        if(logo && typeof(logo) !== 'string') {
```

Comment:
I'm unclear that we should store the logo as a `string`. See comment in the issue.